app/components/school_recruited_transition_component.html.erb

```diff
@@ -1,5 +1,9 @@
 <%= govuk_notification_banner(title: "Important") do |banner| %>
-  <% banner.add_heading(text: "#{delivery_partner.name} has confirmed they will be delivering an induction programme to your early career teachers, starting in #{cohort.display_name}.") %>
-  <p>If this is a mistake, <%= govuk_link_to "report it now", challenge_partnership_path(partnership: partnership.id) %></p>
-  <p>If you do not report this by <%= partnership.challenge_deadline.to_s(:govuk) %>, your programme and next steps will change</p>
+  <% banner.add_heading(text: "#{delivery_partner.name}, with #{lead_provider.name}, has confirmed your school.") %>
+  <p>This means they will be delivering an induction programme to your early career teachers, starting in
+    <%= cohort.start_year %>.</p>
+  <p>If this is a
+    mistake, <%= govuk_link_to "report it now", challenge_partnership_path(partnership: partnership.id) %>.</p>
+  <p>If you do not report this by <%= partnership.challenge_deadline.to_s(:govuk) %>, your programme and next steps will
+    change.</p>
```

Comment:
I don't think this change shows up in percy. Can we add a snapshot that shows the banner?